sycl/include/CL/sycl/accessor.hpp

```diff
@@ -292,6 +293,12 @@ class image_accessor
   static_assert(Dimensions > 0 && Dimensions <= 3,
                 "Dimensions can be 1/2/3 for image accessor.");
 
+  template <info::device param>
+  void check_device_feature_supported(const device &Device) {
```

Comment:
Please rename to checkDeviceFeatureSupported style.

Reply:
Done